salt/pillar/makostack.py

```diff
@@ -35,12 +39,12 @@
 ~~~~~~~~~~~~~~~~~~
 
 This is the simplest option, you just need to set the path to your single
-MakoStack config file like below:
+MakoStack config file as shown below:
 
 .. code:: yaml
 
     ext_pillar:
-      - makostack: /path/to/stack.cfg
+    - makostack: /path/to/stack.cfg
```

Comment:
the ext_pillar documentation here https://docs.saltproject.io/en/latest/topics/development/modules/external_pillars.html  includes the spaces. I think we should keep things consistent